examples/src/content/jcr_root/apps/core-components-examples/clientlibs/clientlib-fonts/css.txt

```diff
@@ -18,3 +18,5 @@
 
 vendor/font-awesome/fontawesome.css
 vendor/font-awesome/solid.css
+vendor/google-webfonts/source-code-pro.css
```

Comment:
@bpauli I wouldn't put these under "google-webfonts" as they are open source Adobe fonts (https://github.com/adobe-fonts) and no longer served via Google. We could just move them under "adobe-fonts" perhaps.